hdr/sqlite_modern_cpp.h

```diff
@@ -405,41 +405,61 @@ namespace sqlite {
 		);
 	}
 	
+	enum class Flags {
+		OPEN_READONLY = SQLITE_OPEN_READONLY,
+		OPEN_READWRITE = SQLITE_OPEN_READWRITE,
+		OPEN_CREATE = SQLITE_OPEN_CREATE,
+		OPEN_NOMUTEX = SQLITE_OPEN_NOMUTEX,
+		OPEN_FULLMUTEX = SQLITE_OPEN_FULLMUTEX,
+		OPEN_SHAREDCACHE = SQLITE_OPEN_SHAREDCACHE,
+		OPEN_PRIVATECACH = SQLITE_OPEN_PRIVATECACHE,
+		OPEN_URI = SQLITE_OPEN_URI
+	};
```

Comment:
@zauguin `OpenFlags::READONLY` looks better, I'm for it 😉 
BTW do you think we should make a new release after this PR?